paper-drawer-panel.html

```diff
@@ -437,7 +437,6 @@
           this.selected = this.defaultSelected;
         }
 
-        this.setScrollDirection(this._swipeAllowed() ? 'y' : 'all');
```

Comment:
Ah ok, thanks for the explanation. I'll add it to the externs then and add it back here.